### PR TITLE
Home Work 17

### PR DESCRIPTION
**Pull request by KostiantynVorobiov**

### CODE REVIEW

File: src/main/java/core/basesyntax/ChangeToCar.java [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод, который принимает вес авто
  * и возвращает тип авто в зависимости от веса (если weight меньше 1500, то CAR, если больше, то
  * TRUCK). Метод должен принимать Integer и возвращать AutoType</p>
  */
-public class ChangeToCar {
+public class ChangeToCar implements Function<Integer, AutoType> {
+    private static final int WEIGHT = 1500;
+
+    @Override
+    public AutoType apply(Integer integer) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/ChangeToCar.java:14]: the variable name is not informative

File: src/main/java/core/basesyntax/CompareTwoAuto.java [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для сравнения мощности
  * двигателей у двух авто (enginePower) и возврата разницы между ними.
  * Метод должен принимать Integer.</p>
  */
-public class CompareTwoAuto {
+public class CompareTwoAuto implements BinaryOperator<Integer> {
+    @Override
+    public Integer apply(Integer enginePower, Integer enginePower2) {
+        return enginePower - enginePower2;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/CompareTwoAuto.java:13]: https://mate-academy.github.io/jv-program-fulltime/code_style_requirements.html

please see the very first point here
also, it is recommended to read the whole doc

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/ChangeToCar.java (hunk 1 above)
Comment (KostiantynVorobiov) [reply src/main/java/core/basesyntax/ChangeToCar.java:14]: Done

File: src/main/java/core/basesyntax/CompareTwoAuto.java (hunk 2 above)
Comment (KostiantynVorobiov) [reply src/main/java/core/basesyntax/CompareTwoAuto.java:13]: Done

**Pull request closed without merging**